test/verify/check-sosreport

```diff
@@ -35,20 +35,18 @@ class TestSOS(MachineCase):
         b = self.browser
         m = self.machine
 
-        # TODO - Sos 4 has moved config to /etc/sos/sos.conf and uses a different format.
-        # But we can't use that because of https://bugzilla.redhat.com/show_bug.cgi?id=1882015
-        self.restore_file("/etc/sos.conf")
-        m.write("/etc/sos.conf",
```

Comment:
We might still need to support SOS 3 in our tests, no? Or has everyone upgraded to version 4?

Reply:
Apparently not, nice!

Reply:
Lets see what has sos4:

- [x] rhel-8-3
- [x] ubuntu-2004
- [x] ubuntu-stable
- [ ] centos-8-stream - has sos 3.9
- [ ] debian-stable - has sos 3.6
- [x] debian-testing
- [x] fedora-32
- [x] fedora-33
- [x] fedora-coreos (does not have sos, so is fine)
- [x] rhel-8-4